fiftyone/operators/delegated.py

```
@@ -516,21 +608,181 @@ def execute_operation(
+                        reason = (
+                            "Operation marked as FAILED externally"
+                            if not err
+                            or "marked as failed" in str(err).lower()
+                            else f"Operation FAILED (detected by monitor): {err}"
+                        )
+                        self._terminate_child_process(
+                            child_process, operation.id, reason
+                        )
+                        result = ExecutionResult(error=reason)
+                        break
+                except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)